src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,76 @@
+
+Reliance on miner honesty only works if there is a cost associated with mining and unpredictability about which specific node submits the next block. These conditions are met by imposing proof-of-work (PoW). To be eligible to submit a block of transactions, a miner must solve an arbitrary computational puzzle faster than any other miner. Solving this puzzle creates competition between miners and costs in the form of energy expenditure. To successfully defraud the blockchain, a dishonest miner would have to consistently win the proof-of-work race, which is very unlikely and prohibitively expensive.
+
+Ethereum has used proof-of-work since genesis. Migrating off of proof-of-work has always been a fundamental goal of Ethereum. Still, it has been philosophically and technologically challenging because the viable alternatives all required to compromise Ethereum's core principles.
+
+## Proof-of-work energy expenditure {#proof-of-work}
+
+Proof-of-work is a robust way to secure against dishonest changes to the blockchain, but it is problematic for several reasons. Since the right to mine a block requires solving a computational puzzle, miners can increase their odds of success by investing in more powerful hardware. These incentives cause an arms race with miners acquiring increasingly power-hungry mining equipment. Ethereum's proof-of-work protocol currently consumes as much energy as a medium-sized country.
+
+## Proof-of-stake {#proof-of-stake}
+
+A greener future for Ethereum is already being built in the form of a **proof-of-stake (PoS)** chain. Under proof-of-stake, arbitrary puzzle-solving is unnecessary. Removing puzzle-solving drastically reduces the energy expenditure required to secure the network. Miners get replaced by validators who perform the same function except that instead of expending their assets up-front in the form of computational work, they stake ETH as collateral against dishonest behavior. If the validator's node is non-responsive or a fraudulent block gets submitted to the chain, the staked assets can be "slashed", strongly incentivizing honesty and securing the network.
+
+Similarly to proof-of-work, to maintain a fraudulent blockchain, a validator would require 51% of the total ETH staked in the network. However, unlike proof-of-work, consensus is not based on the longest chain, but a mechanism known as ["Casper"](https://arxiv.org/abs/1710.09437). Migrating from proof-of-work to proof-of-stake eliminates the need to expend energy on arbitrary computations.
```

Comment:
This is all true, but at the risk of getting in over my head, not sure the notes about Casper are relevant here. Casper GHOST votes, or greediest heaviest observable subtree, is part of the consensus rules, but I don't see how that saves any energy over longest-chain... the energy benefit comes from a new way to select who gets to propose a block.
In PoS, we have collateral, so users can be randomized to propose new blocks without needing PoW to account for Sybil resistance. I don't believe the chain selection rules have much to do with this.

I think a powerful thing we could note here would be that an attack on the network with PoW only costs the energy to spin up a ton of miners (ASICs/GPUs can be rented and spun up for a quick attack)... in PoS you risk losing your entire stake if the attack is large enough.

Reply:
fair point - the references to casper were a bit of a hangover from a prior, excessively detailed version. Agree with the deletion. 

Also like the idea of emphasising the additional security benefits of PoS. It's quite a big topic though and hard to summarize concisely. Maybe best just to link out to the comprehensive explanations on vitalik.ca?